### DISCUSSION TITLE
Rain wallpaper cropping background

### DISCUSSION THREAD

**Discussion by Nitrowing** [rocksdanister/lively#2497] [General]
**Describe the bug**

**To Reproduce**
1. Go to 'Rain'
2. Click on 'Choose wallpaper'
3. Scroll down to 'OK'

**Expected behavior**
The 3840x2160 wallpaper appears on my desktop

**Screenshots/Video**
If applicable, add screenshots/video to help explain your problem.

**Desktop (please complete the following information):**
 - Latest W11
- Rain

**Additional context**
If I add a border to my UHD pic so it becomes 4130x2830 then the pic is full sized on the desktop, the program is cropping and zooming the image.
![bigger](https://github.com/user-attachments/assets/490186ae-1971-4540-949f-96f47fdd0514)
![zoomed](https://github.com/user-attachments/assets/67242abe-1a2a-49d7-85c1-015146f66abe)
[lively_log_20241129_230529.zip](https://github.com/user-attachments/files/17963435/lively_log_20241129_230529.zip)

**Log file (Important)**
Share the diagnostic files by:
Open Lively settings (Gear icon) top-right -> System -> Log File -> Save
or 
Navigate to Appdata folder `C:\Users\<UserName>\AppData\Local\Lively Wallpaper\logs`

**Comment by rocksdanister** [comment]
https://github.com/user-attachments/assets/baee6e51-7b23-42eb-86da-ba6fbc1282a0

**Comment by Nitrowing** [comment]
![stretch fill](https://github.com/user-attachments/assets/6b2546ee-2931-4594-9423-9aa5388e25b8)

I only have the options for Stretch or Fill

**Comment by rocksdanister** [comment]
This is an html wallpaper, the scaling settings are written by the wallpaper creator.

In the case of rain it's complicated to add new fit settings since its done on the shader side, better edit the image instead.. its not suppose to fully show the background due to parallax offsets.

If you still want to change things, you will have to edit the rain project:
https://github.com/rocksdanister/rain
